### DISCUSSION TITLE
[react-markdown] demo page code

### DISCUSSION THREAD

**Discussion by rmonnier9** [remarkjs/remark#749] [Q&A] [answered]
Hi !

I would like to know if the code of this demo page is available somewhere ?
(https://remarkjs.github.io/react-markdown/)

It would be an easy way to try the plugin in a local mode before using it.

Thanks !

**Comment by wooorm** [comment, marked as answer]
On the `website` branch: https://github.com/remarkjs/react-markdown/tree/website.
There are also CodeSandbox starters linked from issues (click new issue, copy the link, or search on code sandbox for react-markdown)

**Comment by ChristianMurphy** [comment]
The source for the demo is available at https://github.com/remarkjs/react-markdown/tree/website
For trying out plugins codesandbox can be a quick and easy option as well https://codesandbox.io/s/react-markdown-debug-9n4eg
